pkgs/tools/admin/awsweeper/default.nix

```diff
@@ -4,19 +4,21 @@ buildGoModule rec {
   pname = "awsweeper";
   version = "0.7.0";
 
+  patches = [ ./gogenerate.patch ];
```

Comment:
Not sure if checking in a 34500 line (1.72 MB) patch is a good idea?

Reply:
Certainly not.

Reply:
Is it possible to add this to a gist?

Reply:
Too big for a gist, I'll just store it in a github repo.

Reply:
Stored and downloaded dynamically now.